tools/sv-report

```diff
@@ -115,20 +116,26 @@ def formatSrc(ifile, ofile):
 
     formatter = HtmlFormatter(
         full=False, linenos=True, anchorlinenos=True, lineanchors='l')
-    with open(ifile, 'rb') as f:
+
+    os.makedirs(os.path.dirname(ofile), exist_ok=True)
+
+    with open(ofile, 'w') as out:
+        try:
+            f = open(ifile, 'rb')
+        except IOError:
+            out.write('Error when opening source file')
```

Comment:
Looks great in general, thanks!

But could we also log the erroneous file name? i.e `f'Error when opening {ifile}'`

Reply:
Sure thing, will add it